book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, self, person)
```

Comment:
- [ ]  the paramètre is not correct, the rental is for the book, not a person we already initialize the person to take the rental.